py/client/pydeephaven/session.py

```diff
@@ -572,6 +550,42 @@ def bind_table(self, name: str, table: Table) -> None:
         """
         self.console_service.bind_table(table=table, variable_name=name)
 
+    def publish(self, source_ticket: Ticket, result_ticket: Ticket) -> None:
+        """ Publishes a source ticket to the result ticket.
+
+        This is low-level method that should be used mostly to publish non-Table server objects that are previously
+        fetched from the server. The source ticket represents the previously fetched server object to be published, and
+        the result ticket, which should normally be a SharedTicket, is the ticket to publish to. The result ticket can
+        then be fetched by other sessions to access the object as long as the object is not released. This method is
+        used together with the :meth:`.fetch` method to share server objects between sessions.
```

Comment:
Does it work for `Table`?  Is there a reason not to use it for `Table`?  If it is fine for table, I would remove the `non-Table` stuff since I'm wondering why not tables.

Reply:
It does but only return s an export ticket without the required table metadata needed for the Table constructor.

Reply:
Maybe say that.  As a user, the doc feels somehow imprecise when it says things like "mostly".  Users really want to know exactly what it is doing so that they know if they are using the right method and using it correctly.